readme.md

```diff
@@ -597,7 +598,7 @@ Or if you are on [openshift](https://openshift.com):
 </dependency>
 ```
 
-###### Annotation less
+#### Halkyon Component CRD and  Annotationless configuration
 For Spring Boot applications (i.e. at least one project class is annotated with `@SpringBootApplication`) all you need to do, is adding one of the starters (`io.dekorate:kubernetes-spring-starter` or `io.dekorate:openshift-spring-starter`) to the classpath. No need to specify an additional annotation.
```

Comment:
This text starting from `For Spring Boot applications ...` has nothing to do with the title `Halkyon Component CRD & Annotationless configuration`

Reply:
There is something to do because the annotation less mode is only supported for spring boot applications so far. We use `@SpringBootApplication` to trigger the whole process.

Reply:
Then I suggest that you explain/cover that text between the Title `Annotation less` and `Halkyon Component ...` and where you explain what `Annotationless` is/means, how it could be/should be used, How overidding is working, ...